### ISSUE TITLE
Multiple Image Field

### ISSUE THREAD

**Issue by inluxc** [tzyganu/UMC1.9#93]
It would be nice to have multiple images fields like the product section.

**Comment by tzyganu** [comment]
@inluxc I agree with you. It would be nice to have it. Unfortunately it's not that easy.  
I will leave this issue open, just in case I get a good idea or some spare time in the future to do it.  
Also this is up for grabs. If someone from the community wants to work on this, I would really appreciate it. (and I'm sure others will appreciate it as well)

**Comment by alexdrupal** [comment]
Interested in this feature as well

**Comment by pljspahn** [comment]
This would be nice. When adding a Field / Attribute, maybe there can be an additional radio option that makes it a gallery which would then add the attribute based on what Magento does:

https://github.com/OpenMage/magento-mirror/blob/d6654602d3f3b612da41d576ae65ebc4241abf25/app/code/core/Mage/Catalog/sql/catalog_setup/mysql4-upgrade-0.7.7-0.7.8.php#L31

I am happy to help some however I can if anyone else is interesting in doing a PR.
